srcpkgs/cage/template

```diff
@@ -1,25 +1,25 @@
 # Template file for 'cage'
 pkgname=cage
 version=0.1.2.1
-revision=2
+revision=3
 build_style=meson
 configure_args="$(vopt_bool xwayland xwayland)"
 hostmakedepends="pkg-config wayland-devel scdoc"
 makedepends="wlroots-devel"
-depends="$(vopt_if xwayland xorg-server-xwayland) $(vopt_if elogind elogind)"
+depends="$(vopt_if xwayland xorg-server-xwayland)"
 short_desc="Kiosk compositor for Wayland"
 maintainer="Illia Shestakov <ishestakov@airmail.cc>"
 license="MIT"
 homepage="https://www.hjdskes.nl/projects/cage/"
 distfiles="https://github.com/Hjdskes/cage/archive/v${version}.tar.gz"
 checksum=38a3e3968f00cc58fe1d9448e972cfac7d1efa30c48699f09032f264101a55ac
 
-build_options="elogind xwayland"
-build_options_default="xwayland"
+build_options="setuid xwayland"
```

Comment:
Please add a description to `common/options.description`.

Reply:
Added